docs/feature-store/sources-targets.md

```diff
@@ -199,6 +199,26 @@ For example:
 Disable partitioning with:
 - `ParquetTarget(partitioned=False)`
 
+## Snowflake target
+
+```python
+profile = DatastoreProfileSnowflakeTarget(name="profile-name",bootstrap_servers="localhost", topic="topic_name")
+target = SnowflakeTarget(path="ds://profile-name")
+```
+
+`DatastoreProfileKafkaTarget` class parameters:
```

Comment:
do you mean `SnowflakeTarget` ?

Reply:
@tomerm-iguazio - if there is no datastore profile - should that line be "SnowflakeTarget parameters"?